lib/hal/audio.h

```diff
@@ -31,7 +31,9 @@ typedef struct
 	int                      numChannels;
 } AC97_DEVICE  __attribute__ ((aligned (8)));
 
-// note that I currently ignore sampleSizeInBits and numChannels.  They
+// The XAudio API is only supposed to be used as a backend. Using SDL2 for 
+// audio playback should be prefered for applications.
+// note that currently sampleSizeInBits and numChannels are ignored. They
```

Comment:
nit: Capitalize "note"

Reply:
nit: "They" at end of line is equally unfortunate (I did not check how the comment continues, but this looks odd).

Also not sure if we do this in nxdk, but I personally also usually use markdown for code stuff in comments, so I'd use:

```
[..] currently `sampleSizeInBits` and [..]
```

Reply:
Applied both nits and the markdown-code-comment idea.